src/components/GameList/GameList.tsx

```diff
@@ -320,12 +275,21 @@ export class GameList extends React.PureComponent<GameListProps, GameListState>
                     currentSort={this.state.sort_order}
                     player={this.props.player}
                     lineSummaryMode={this.props.lineSummaryMode}
+                    onFirstClock={() => this.onFirstClock()}
                 ></LineSummaryTable>
             );
         } else {
             return MiniGobanList(games, !!this.props.namesByGobans, this.props.miniGobanProps);
         }
     }
+
+    onFirstClock() {
+        // Now that (more) gobans have their first clock, force render() to be
+        // called again so that the default sort (by clock) works.
+        this.setState({
+            force_render: this.state.force_render + 1,
+        });
+    }
```

Comment:
Or maybe this function should just be called `forceRender()`. (The callbacks would still refer to it as `onFirstClock()`.)

Reply:
Either way, it's easy to trace through this way, on this flip side `forceRender`  describes exactly what it does, both are good imho.